orderer/common/multichannel/chainsupport.go

```diff
@@ -159,8 +159,8 @@ func (cs *ChainSupport) ProposeConfigUpdate(configtx *cb.Envelope) (*cb.ConfigEn
 		return nil, errors.New("new config is missing orderer group")
 	}
 
-	if err = cs.ValidateConsensusMetadata(oldOrdererConfig, newOrdererConfig, false); err != nil {
-		return nil, errors.Wrap(err, "consensus metadata update for channel config update is invalid")
+	if err = cs.ValidateConsensusMetadata(oldMetadata, newMetadata, false); err != nil {
```

Comment:
I think you forgot to update the variable declarations as well, these variable don't exist yet, this is why CI is failing